pkg/provisioner/demo/demo.go

```diff
@@ -250,38 +248,13 @@ func (p *demoProvisioner) Provision(_ provisioner.ProvisionData, _ bool) (result
 // multiple times, and should return true for its dirty flag until the
 // deprovisioning operation is completed.
 func (p *demoProvisioner) Deprovision(_ bool) (result provisioner.Result, err error) {
-	hostName := p.objectMeta.Name
-	switch hostName {
-	default:
-		return result, nil
-	}
-
-	// p.log.Info("ensuring host is removed")
-
-	// result.RequeueAfter = deprovisionRequeueDelay
+	p.log.Info("deprovisioning host")
+	return result, nil
 
 	// // NOTE(dhellmann): In order to simulate a multi-step process,
 	// // modify some of the status data structures. This is likely not
 	// // necessary once we really have Demo doing the deprovisioning
 	// // and we can monitor it's status.
```

Comment:
No point leaving this comment, which refers to the deleted code.

Reply:
Done